circuit/src/verifier/transcript/mod.rs

```diff
@@ -647,7 +647,7 @@ mod test {
 		type FloorPlanner = SimpleFloorPlanner;
 
 		fn without_witnesses(&self) -> Self {
-			self.clone()
+			Self { reader: Vec::new() }
```

Comment:
This should be handled the same way as:
https://github.com/eigen-trust/protocol/blob/master/circuit/src/verifier/transcript/mod.rs#L698-L714